extension/package.json

```diff
@@ -1158,42 +1163,42 @@
         {
           "command": "dvc.views.experiments.applyExperiment",
           "group": "inline@1",
-          "when": "view == dvc.views.experimentsTree && dvc.commands.available && viewItem == experiment && !dvc.experiment.running"
+          "when": "view == dvc.views.experimentsTree && dvc.commands.available && viewItem == experiment && !dvc.experiment.running.workspace"
```

Comment:
[F] `viewItem == experiment` means none of these actions will be available for running experiments.

Reply:
I'm going to add stop inline as a follow up